### ISSUE TITLE
Ability to configure static import sorting to match behaviour of IntelliJ IDEA

### ISSUE THREAD

**Issue by hakanai** [diffplug/spotless#879]
If you have the following configuration:

```groovy
importOrder('java', 'javax', '', 'acme', 'com.acme', '\\#')
```

Spotless will sort the static imports like this:

```java
import static com.github.tomakehurst.wiremock.client.WireMock.*;
import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
import static org.hamcrest.Matchers.*;
```

But IntelliJ IDEA will sort the static imports like this:

```java
import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
import static com.github.tomakehurst.wiremock.client.WireMock.*;
import static org.hamcrest.Matchers.*;
```

This causes constant fighting between IDEA and Spotless so it would be nice if we could configure spotless to say that the wildcard always comes last.

**Comment by nedtwigg** [comment]
PR welcome :) https://github.com/diffplug/spotless/blob/main/lib/src/main/java/com/diffplug/spotless/java/ImportSorter.java

Maybe add a `boolean wildcardsLast = false`, which can be set to true in order to get the IntellIJ behavior? Related #200

**Comment by sbilello** [comment]
Is there a way to use the same java style in intellij in spotless? https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml

**Comment by hakanai** [comment]
@sbilello that's a separate issue but yeah that would probably be ideal for us if it were possible. I believe we were waiting on JetBrains breaking out their reformatting code into a library so that it could be used for that?

**Comment by hakanai** [comment]
I'm having a look at this at the moment.

I think I have sorted out the way it would work in the DSL:

```
importOrder('java', 'javax', 'com.acme', '').wildcardsLast()
```

So I wrote the tests. The test for default ordering passes, the test for `wildcardsLast` fails. Good, so then I wrote all the code to get that `wildcardsLast` value into `ImportSorterImpl` and am trying to figure out where in the code it's actually doing the ordering of wildcard vs. non-wildcard.
